### ISSUE TITLE
Re-simulation of CIFAR 100 results with different pre-processing using wrn

### ISSUE THREAD

**Issue by revilokeb** [ibmua/Breaking-Cifar#1]
@ibmua I have run a few simulations based on the code and the model https://github.com/ibmua/Breaking-Cifar/blob/master/models/wide-res-restructured-pooled-actually.lua which seems to be your best performing based on your comments in https://github.com/ibmua/Breaking-Cifar/blob/master/notebooks/visualize.ipynb

Depending on the choice of preprocessing I am obtaining the following errors:
- CIFAR 100 whitened (https://yadi.sk/d/em4b0FMgrnqxy): ~20.2%
- CIFAR 100 combined (https://github.com/ibmua/Breaking-Cifar/blob/master/datasets/cifar100_combined.t7.tar.gz): ~18.6%
- CIFAR 100 std_mean using your little script on CIFAR 100 combined: ~18.8%

I have run simulation for each for around 250 epochs, the above numbers are "eye-balling averages" of the last ~30 epochs.

Have you run simulations on mean / std adjusted dataset and have your results been even better?

**Comment by ibmua** [comment]
No, I actually ran with std & mean adjusted only on smaller tests and other models, like the Zagoruyko's original WRN. Back there all seemed to be pretty dependent on random, but on average adjusting std &, probably even more importantly, mean, gave a bit of improvement. I'm not sure why, though. Whether it also gives results on huge models --- I honestly don't know. But as far as I know, that's the preprocessing that many popular nets use, ResNet being one of 'em. My guess is that they likely have measured impact of this preprocessing.

If you'll be running more big model tests with different preprocessing schemes, please, tell if the trend continues, or reverses.

**Comment by revilokeb** [comment]
Ok thanks for the info! I will update if I am finding anything interesting.
